velox/vector/FunctionVector.h

```diff
@@ -15,14 +15,11 @@
  */
 #pragma once
 
+#include "velox/expression/EvalCtx.h"
```

Comment:
Looks like this introduces circular dependency: vectors depend expressions and expressions depend on vectors. We need to figure out how to break that.

Reply:
Maybe move ErrorVectorPtr to FlatVector.h.

```
  using ErrorVector = FlatVector<std::shared_ptr<void>>;
  using ErrorVectorPtr = std::shared_ptr<ErrorVector>;
```